docs/source/test_network.md

```diff
@@ -239,34 +239,30 @@ chaincode on the channel using the following command:
 ```
 ./network.sh deployCC
 ```
-The `deployCC` subcommand will install the **fabcar** chaincode on
+The `deployCC` subcommand will install the **asset-transfer (basic)** chaincode on
 ``peer0.org1.example.com`` and ``peer0.org2.example.com`` and then deploy
 the chaincode on the channel specified using the channel flag (or `mychannel`
 if no channel is specified).  If you are deploying a chaincode for the first
 time, the script will install the chaincode dependencies. By default, The script
-installs the Go version of the fabcar chaincode. However, you can use the
+installs the Go version of the asset-transfer (basic) chaincode. However, you can use the
 language flag, `-l`, to install the Java or javascript versions of the chaincode.
```

Comment:
Asset basic is not yet available in Java.

Reply:
Reference to Java removed.